### DISCUSSION TITLE
Fans not spinning on low RPM in FanControl but with BIOS control they do

### DISCUSSION THREAD

**Discussion by DonFurkan** [Rem0o/FanControl.Releases#2243] [Q&A]
Hello, im having issues with my Case Fans not spinning at 'low' RPMs or %PWM in FanControl.

My Setup: Windows 10
CPU&MB; Ryzen 5 5600x, Msi b550 a pro
CPU Cooler: Thermalright Frost Spirit 140
Case Fans: 3x Arctic P12 Max, 2x Arctic P14 PWM
CPU Cooler: Thermalright Frost Spirit 140

When i control my case fans in my BIOS i can set them as low as about 9%PWM (P12) or 15%PWM (P14) and they start spinning.
However when i try to setup curves or manual control my case fans in FanControl they start at about or above 20% (P12) or even 30%(P14). The CPU cooler works fine i can set it as low as 4% in the BIOS and FanControl and they spin.

Im new to GitHub but i searched in discussions and didn't find anything similar. I tried the recommendations from the FAQ of turning off smart fan control in the BIOS and also tried DC mode but it didnt help.
Im also quite new to fancontrol but i think i got the hang of it except for this odd (i think) behavior everything works fine and i love this program and would like to be able to use this program with the low fans speeds that i can set in the BIOS.

In the Screenshot you can see that if i manual control the fans to 10%PWM (p12) they don't spin but they do spin if i set them to 10%PWM in the BIOS.(Same for the p14 just higher %)
![Screenshot 2024-01-08 174344](https://github.com/Rem0o/FanControl.Releases/assets/155996441/66b9e018-f3dc-4778-88f2-eb04006fb2d1)

**Comment by Rem0o** [comment]
Expand the cards.   You probably went with the assisted setup?   When detecting start and stop %, fan control will set a generous margin of error for the start and stop % to be on the safe side. You can tweak those up or down manually.

**Reply by DonFurkan** [reply]
Wow thanks for the quick response and fix, yeah i used the assisted setup but didn't notice the start and stop values.
Now it works perfect!
